csv.h

```diff
@@ -753,30 +787,26 @@ namespace io{
                 }
         };
 
-        template<char sep>
-        struct no_quote_escape{
-                static const char*find_next_column_end(const char*col_begin){
-                        while(*col_begin != sep && *col_begin != '\0')
-                                ++col_begin;
-                        return col_begin;
-                }
-
-                static void unescape(char*&, char*&){
+        
 
-                }
-        };
-
-        template<char sep, char quote>
+        template<char sep, char quote, bool null_terminated = true>
         struct double_quote_escape{
+				static bool hasQuote() {
+					return true;
+				}
+				static char getQuoteChar() {
+					return quote;
```

Comment:
This reduces the flexibility given by the policy system. For example, before it was possible to write a policy that parses a,b\,c,d as three columns with values "a", "b,c" and "d".

I doubt anybody used this freedom, but it is an introduced restriction.